content/en/data_security/guide/tls_ciphers_deprecation.md

```diff
@@ -4,7 +4,9 @@
 
 As part of its ongoing commitment to the security and protection of its customer's data, Datadog is rolling out a more modern cryptographic engine across its systems which imposes some changes to the configurations it can accept.
 
-Beginning **April 1st, 2024**, Datadog is disabling support for the following cipher suites across its public-facing applications. If you use unsupported clients to connect to Datadog after the older protocols are disabled, you will receive connection error messages.
+Beginning **April 1st, 2024**, Datadog disabled support for the following cipher suites across its public-facing applications. If you use unsupported clients to connect to Datadog after the older protocols are disabled, you will receive connection error messages.
+
+Starting **July 1st, 2024**, Datadog will force HTTP to HTTPS redirection for all our intake endpoints, using an HTTP 308 response code. This change is requierd because when maintaining compatibility with legacy API clients, some of our data intake endpoints accept plain HTTP payloads. Ensure that your application clients can follow these redirection codes, or update their configuration to use HTTPS. Otherwise, you might receive connection error messages.
```

Comment:
```suggestion
Starting **July 1st, 2024**, Datadog will force HTTP to HTTPS redirection for all intake endpoints, using an HTTP 308 response code. This change is necessary as some intake endpoints still accept plain HTTP payloads to maintain compatibility with older API clients. Ensure that your application clients can follow these redirection codes, or update their configuration to utilize HTTPS. Otherwise, you might receive connection error messages.
```